src/index.ts

```diff
@@ -47,11 +62,63 @@ const isNick = (message: Message): boolean => {
 const isDaniel = (message: Message): boolean => {
   return (
     message.content.replace(/\s/g, "").toLowerCase().includes("daniel") ||
+    message.content.replace(/\s/g, "").toLowerCase().includes("dan") ||
     message.author.id === "731236961439973458" ||
+    message.content.includes("D A N") ||
     message.content.includes("D A N I E L")
   );
 };
 
+const isBen = (message: Message): boolean => {
+  return (
+    message.content.replace(/\s/g, "").toLowerCase().includes("ben") ||
+    message.content.replace(/\s/g, "").toLowerCase().includes("benjamin") ||
+    message.author.id === "664240791861985320" ||
+    message.content.includes("B E N J A M I N") ||
+    message.content.includes("BEN")
```

Comment:
```suggestion
    message.content.includes("B E N")
```
If these are the unicode versions (for the regional indicator emoji) they need to be space separated.